quence.lua

```diff
@@ -900,10 +898,8 @@ function sync_tracks()
 end
 
 function clear_all_notes()
+    engine.noteKillAll()
```

Comment:
Really should've put a comment in here, but; we don't have to loop over anything - just kill all.